pkg/debug/start_debug.go

```diff
@@ -97,6 +97,12 @@ func startDebug(context *k8sutil.Context, clusterNamespace, deploymentName, alte
 		return err
 	}
 
+	pod, err := k8sutil.WaitForPodToRun(context, clusterNamespace, labelSelector)
```

Comment:
yes, it is using the same label ad the main pod as debug pod also has the same labels as main pod. So, I don't see any other unique label from main pod as debug pod is copy or main pod only few specs are changes.

Reply:
Ok, I guess if they use the same label then this will wait for the terminating pod to go away and the debug pod to be in running state?

Reply:
it will only wait for the debug pod to be running and not the main pod to be deleted but I always noticed the before debug pod has started main pod was deleted. So, I think we are good here.

Reply:
@travisn ^^ if above sounds okay, I can go ahead and merge the pr